src/Processors/QtiV2/In/Interactions/TextEntryInteractionMapper.php

```diff
@@ -25,6 +47,35 @@ public function getQuestionType()
         return $closetext;
     }
 
+    public function getOption($key, $defaultValue = null) {
+        $value = $defaultValue;
+        if (isset($this->options[$key])) {
+            $value = $this->options[$key];
+        }
+
+        return $value;
+    }
+
+    public function isUsingExpectedLengthAsMaxCharLimit() {
+        return (bool)$this->getOption('expected_length_as_max_char_limit', false);
```

Comment:
Will add `const` values for the defaults here.

Reply:
I think a better way to handle this is to declare the defaults inside $options. I.e.

```
protected $options = [
    'expected_length_as_response_width' => true,
    'expected_length_as_max_char_limit' => false,
    ....
];
```
and have the getter/isser just call getOption without passing in a default. The advantage of this is we do not have to coerce getter/isser to a particular type.

Of course you can still use the setters to override settings.